src/sql/workbench/contrib/query/browser/queryResultsView.ts

```diff
@@ -201,7 +201,7 @@ export class QueryResultsView extends Disposable {
 	private hasResults(runner: QueryRunner): boolean {
 		let hasResults = false;
 		for (const batch of runner.batchSets) {
-			if (batch.resultSetSummaries.length > 0) {
+			if (batch.resultSetSummaries?.length > 0) {
```

Comment:
Please update the typings too if this can truly be undefined. Is undefined something that happens when there's an error?